k8s/apis/model/v1/common_types.go

```diff
@@ -110,6 +110,7 @@ type CatalogSpec struct {
 	Name   string `json:"name"`
 	// +kubebuilder:pruning:PreserveUnknownFields
 	// +kubebuilder:validation:Schemaless
+	Metadata   map[string]string    `json:"metadata,omitempty"`
```

Comment:
We have kubebuilder marker here. And I believe this is for Properties (runtime.RawExtension). So correct the order.

Reply:
We don't such markers for map[string]string